src/cdk/v2/destinations/bluecore/procWorkflow.yaml

```diff
@@ -55,6 +55,21 @@ steps:
             $.removeUndefinedNullValuesAndEmptyObjectArray(newPayload)
           )[];
 
+  - name: handleSubscriptionEvent
+    condition: $.outputs.messageType === {{$.EventType.TRACK}} && $.isSubscriptionEvent(.message.event)
+    steps:
+      - name: validateInput
+        description: Additional validation for Track events
+        template: |
+          $.assert(.message.event, "event_name could not be mapped. Aborting.")
```

Comment:
Will this condition ever occur since `isSubscriptionEvent` will already validate the presence of `message.event`